src/interface/substrate_query.js

```diff
@@ -193,6 +193,28 @@ async function getStakingErasValidatorReward(api, eraIndex) {
     }
 }
 
+async function getUnappliedSlashes(api, eraIndex) {
```

Comment:
Can you please rename the function to `getStakingUnappliedSlashes` for consistency with the other functions?

Reply:
Hello,

Done:)

Reply:
Thank you!